src/components/RepoValidation.js

```diff
@@ -7,6 +7,9 @@ import { Workspace } from 'resource-workspace-rcl'
 import { makeStyles } from '@material-ui/core/styles'
 
 import { StoreContext } from '@context/StoreContext'
+//import { AdminContext } from '@context/AdminContext'
```

Comment:
@mandolyte Great work refactoring the ContextProvider code, would it make sense to remove the commented code?

Reply:
Feel free to remove the commented code in your next PR if you so desire.